build-support/native-image/README.md

```diff
@@ -0,0 +1,87 @@
+native-image `META-INF/`
+===========================
+
+**NOTE: currently, these instructions will only work when used from the script in this repo [^1]! Image building in general is currently blocked on https://github.com/oracle/graal/issues/1448.*
+
+*This directory contains special configuration files recognized by the `native-image` tool when embedded in a jar. This embedded configuration allows any user with some version of the Graal VM to fetch the `org.pantsbuild:zinc-compiler` jar and run `native-image -jar` to produce a native executable of the pants zinc wrapper without any additional arguments.*
```

Comment:
This looks stale.

Reply:
This is fresh and very recently updated! The note states that the instructions will only work from a separate, linked script, in which it is described that we currently need to build graal from a forked sha, so it doesn't immediately work with just any native-image.